src/components/LaunchDarkly.js

```diff
@@ -1,9 +1,8 @@
 // @flow
 import React from "react";
-import { Broadcast } from "react-broadcast";
+import {LaunchDarklyProvider} from "./Context";
```

Comment:
Beginning nits - bracket spacing should follow the conventions in the rest of the lib.